.github/workflows/dbip_download.yml

```diff
@@ -0,0 +1,63 @@
+name: Download DBIP data and generate dbip_country.rs
+
+on:
+  workflow_dispatch:
+#  schedule:
+#    - cron: "0 0 3 * *" # Runs at midnight on the 3rd of every month
+
+env:
+  SKIP_EXPORT: 'false'
```

Comment:
There's something you have to specifically set in a `.yml` file (I forget what it is) to make Actions keep going after an error. Normally, any error causes an abort. Therefore, I don't think this is necessary.